README.MD

```diff
@@ -1 +1,47 @@
 # GPostCard
+
+### Project Description
+
+GPostcard is a Web application that can generate postcards based on the text that is on them. After a postcard is generated, it can be sent to any Gmail user. By ticking the location, the user can get on the postcard a representative photo for their current location. Moreover, the user can record a message which will be automatically filled into the text area allowing accessability. 
```

Comment:
There is this PR https://github.com/googleinterns/step230-2020/pull/67 that deals with the README file. Let's focus there on this file.

Reply:
Oops, of course. Sorry!